sources/models/shared-defaults/oci-defaults-docker-resource-limits.toml

```diff
@@ -0,0 +1,3 @@
+[settings.oci-defaults.resource-limits.max-open-files]
```

Comment:
These limits seem really low. We set them significantly higher for [containerd](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/models/shared-defaults/oci-defaults-containerd-cri-resource-limits.toml). I think this is what the current limits are for `docker` but it feels strange to have wildly different limits between the two. We have three options in my mind (in order of my preference):
* Move these limits to align to `containerd` for all docker-based variants to remain consistent across Bottlerocket
* Move only new variants (like `aws-ecs-2`) to higher settings to avoid creating surprises for end users in case they are depending on these very low limits
* Keep `docker` limits lower than `containerd` by default
We are essentially taking the third option with this change as-is. I'd like to at least acknowledge we are doing that if we end up taking that route.

Reply:
Yeah, the limits that we are using now, aligns with what we had for old Bottlerocket versions. But I think so too we can at least raise it for ecs-2 variants keeping same for ecs-1 variant.

Reply:
Keeping the defaults unchanged on ecs-1 variants seems... important.

Reply:
> * Keep `docker` limits lower than `containerd` by default
As discussed offline we are going to keep docker limits lower than Containerd as  it is possible to raise the default limits via task definition for ecs variants.

Reply:
> * Keep `docker` limits lower than `containerd` by default
As discussed offline we are going to keep docker limits lower than Containerd as  it is possible to raise the default limits via task definition for ecs variants.